edit_course/templatetags/editcourse.py

```diff
@@ -34,6 +35,13 @@ def removeurl(model_object, model_name):
     ))
 
 
+@register.simple_tag
+def submittable(entry):
+    if entry.content_type_id == 21:
```

Comment:
There must be a better way to do this. Somehow using `entry.is_submittable` did not work, even though the `is_submittable` is defined in the `LearningObject` class. Entries are not handled as `BaseExercises`, even though the objects would be members of that class.

Reply:
`entry.is_submittable` may also refer to the values in the content cache or some template tag. So it might not directly call the method in the model class (LearningObject). Can you find out which method it really calls and where the function is defined?